Content.Server/_CP14/Explosion/Components/TriggerOnDamageReceivedComponent.cs

```diff
@@ -1,9 +1,7 @@
-using Robust.Shared.GameObjects;
-
 namespace Content.Server.Explosion.Components;
 
 /// <summary>
 /// Triggers when the entity is damaged.
 /// </summary>
 [RegisterComponent]
-public sealed partial class TriggerOnDamageComponent : Component { }
+public sealed partial class TriggerOnDamageReceivedComponent : Component { }
```

Comment:
```suggestion
public sealed partial class CP14TriggerOnDamageReceivedComponent : Component { }
```

also all fork components starts with CP14 prefix too (filename need changes too)